pkg/scheduler/factory.go

```diff
@@ -187,7 +188,7 @@ func (c *Configurator) createFromProvider(providerName string) (*Scheduler, erro
 		prof.Plugins = plugins
 	}
 
-	return c.create([]core.SchedulerExtender{})
+	return c.create(c.extenders)
```

Comment:
Given `extenders` is embedded in `c`, isn't that we can directly call `c.create()`?